src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    private double circleRadius = random.nextDouble(10) + 1;
+    private double firstRightTriangleLeg = random.nextDouble(10) + 1;
+    private double secondRightTriangleLeg = random.nextDouble(10) + 1;
+    private double squareSide = random.nextDouble(10) + 1;
+    private double firstSideIsoscelesTrapezoid = random.nextDouble(10) + 1;
+    private double secondSideIsoscelesTrapezoid = random.nextDouble(10) + 1;
+    private double heightIsoscelesTrapezoid = random.nextDouble(10) + 1;
+    private double rectangleLength = random.nextDouble(10) + 1;
+    private double rectangleWidth = random.nextDouble(10) + 1;
+
+    public Figure getDefaultFigure() {
+        return new Circle("white", 10);
```

Comment:
Use enum and default values
```suggestion
        return new Circle(DEFAULT_COLOR, DEFAULT_RADIUS);
```